tests/run/test_genericclass_exttype.pyx

```diff
@@ -0,0 +1,99 @@
+# mode: run
+# tag: pure3.7
```

Comment:
This isn't needed since it's not a pure Python (`.py`) test but a `.pyx` test, so Python won't see it anyway.